### DISCUSSION TITLE
install for Raspberry pi

### DISCUSSION THREAD

**Discussion by stevecrow74** [AlexandreRouma/SDRPlusPlus#724] [Ideas]
found this online and it helped, maybe add this into the readme. 

sudo apt install libglfw3-dev libglew-dev libairspyhf-dev libiio-dev libad9361-dev libairspy-dev librtlsdr-dev portaudio19-dev libzstd1 libzstd-dev

cd ~/
git clone https://github.com/AlexandreRouma/SDRPlusPlus.git

cd SDRPlusPlus
mkdir build
cd build
cmake .. -DOPT_BUILD_SDRPLAY_SOURCE:BOOL=ON -DOPT_BUILD_NEW_PORTAUDIO_SINK:BOOL=ON -DOPT_BUILD_M17_DECODER:BOOL=ON
make -j4
sudo make install
sudo ldconfig

**Comment by TekMaker** [comment]
Thanks SteveCrow74 I was able to update my article after a succesful build!
